app/partials/errorDetail.html

```diff
@@ -8,14 +12,48 @@
             <th class="action" scope="col">Action</th>
         </tr>
     </thead>
-    <tbody ng-repeat="item in error.errors">
+    <tbody ng-repeat="item in error.errors | paginate:start():end()">
         <tr ng-repeat="(key, value) in item.properties">
             <th class="id" scope="row" rowspan="{{item.properties | keyLength}}" ng-if="$first">{{item.loanNumber || item.lineNumber}}</th>
             <td class="description">{{key | hmdaLabel:error.scope}}</td>
             <td class="value">{{value}}</td>
-            <td class="action" rowspan="{{item.properties | keyLength}}" ng-if="$first">Correct Entry and Resubmit</td>
+            <td class="action" rowspan="{{item.properties | keyLength}}" ng-if="$first">Correct and revalidate</td>
         </tr>
     </tbody>
 </table>
 
+<nav class="pagination" ng-if="error">
+    <button class="btn btn__secondary pagination_prev" ng-class="{btn__disabled: !hasPrev()}" ng-disabled="!hasPrev()" ng-click="onPrev()">
+        <span class="btn_icon__left cf-icon cf-icon-left"></span>
+        Previous
+    </button>
+    <button class="btn btn__secondary pagination_next" ng-class="{btn__disabled: !hasNext()}" ng-disabled="!hasNext()" ng-click="onNext()">
+        Next
+        <span class="btn_icon__right cf-icon cf-icon-right"></span>
+    </button>
+    <div class="pagination_form">
+        <label class="pagination_label bold" for="pagination_current-page">
```

Comment:
in the app LESS/CSS you can always do: 
```css
.pagination_label {
    font-weight: bold;
}
```
That will override just the font-weight property for anything that matches the selector. Now, my guess is that Nathaniel's mockups are being over zealous in styling labels which is affecting the labels in the pagination. I would leave the styling for the pagination alone for the time being since this is an establish pattern in cf that we can follow.